products/compute/api.yaml

```diff
@@ -669,6 +670,26 @@ objects:
         name: 'lastDetachTimestamp'
         description: 'Last dettach timestamp in RFC3339 text format.'
         output: true
+      - !ruby/object:Api::Type::NameValues
+        name: 'labels'
+        description: |
+          Labels to apply to this disk. These can be later modified by the setLabels method.
```

Comment:
I'd get rid of "These can be later modified by the setLabels method" since users of these tools don't need to know that.